src/Libraries/RevitNodes/Elements/InternalUtilities/ElementUtils.cs

```diff
@@ -196,17 +196,24 @@ public static double GetConvertedParameterValue(Autodesk.Revit.DB.Parameter para
             return value;
         }
 
+
         [SupressImportIntoVM]
         internal static UnitType ParameterTypeToUnitType(ParameterType parameterType)
         {
             switch (parameterType)
             {
                 case ParameterType.Length:
                     return UnitType.UT_Length;
+                case ParameterType.ReinforcementLength:
+                    return UnitType.UT_Length;
```

Comment:
It probably should be. You are right. I didn't even know there was a Unit Type named specifically after reinforcement. I am guessing there is a setting in Revit that controls display units for reinforcement independently of other Length and Area settings. Thank you for picking this up.